shared/chat/conversation/messages/set-explode-popup/index.desktop.js

```diff
@@ -19,17 +19,14 @@ const Header = (props: HeaderProps) => (
           fontSize={48}
           style={platformStyles({isElectron: {display: 'block', lineHeight: '28px', maxHeight: 28}})}
         />
-        <Text type="BodySemibold" backgroundMode="Announcements" style={{textAlign: 'center'}}>
+        <Text type="BodySmallSemibold" backgroundMode="Announcements" style={{textAlign: 'center'}}>
           Set a timeout on your messages and watch them
-        </Text>
-        <Text type="BodySemibold" backgroundMode="Announcements">
-          E X P L O D E
+          E&nbsp;&nbsp;&nbsp;X&nbsp;&nbsp;&nbsp;P&nbsp;&nbsp;&nbsp;L&nbsp;&nbsp;&nbsp;O&nbsp;&nbsp;&nbsp;D&nbsp;&nbsp;&nbsp;E.
```

Comment:
Is there a reason we're not using CSS to style this?

Reply:
Sure I can do CSS if that looks messy.

Reply:
might as well leave it, it'll likely not work well on android , this is simpler